api/MediaQueryList.json

```diff
@@ -17,7 +17,7 @@
             "version_added": "6"
           },
           "firefox_android": {
-            "version_added": true
+            "version_added": "6"
```

Comment:
This matches the data for `window.matchMedia` which returns a `MediaQueryList`, so I believe it!